operate/qa/integration-tests/src/test/java/io/camunda/operate/rest/ProcessInstanceRestServiceIT.java

```diff
@@ -131,4 +151,129 @@ public void testListenersRequestWithListenerFilterInvalid() throws Exception {
                 + ListenerType.TASK_LISTENER
                 + "]");
   }
+
+  @Test
+  public void testPostUnsupportedOperationDeleteDecisionRequest() throws Exception {
+    // given
+    final String url = ProcessInstanceRestService.PROCESS_INSTANCE_URL + "/1/operation";
+    final CreateOperationRequestDto request =
+        new CreateOperationRequestDto().setOperationType(OperationType.DELETE_DECISION_DEFINITION);
+
+    // when - then
+    final MvcResult mvcResult = mockMvcManager.postRequest(url, request, 400);
+    assertThat(mvcResult.getResolvedException().getMessage())
+        .contains(
+            "Operation type '%s' is not supported by this endpoint."
+                .formatted(OperationType.DELETE_DECISION_DEFINITION));
+  }
+
+  @Test
+  public void testPostUnsupportedOperationDeleteProcessRequest() throws Exception {
+    // given
+    final String url = ProcessInstanceRestService.PROCESS_INSTANCE_URL + "/1/operation";
```

Comment:
🔧 This is somewhat NABD because the rest of the class uses the same pattern but here we can use the `var` keyword to slightly improve readability

```suggestion
    final var url = ProcessInstanceRestService.PROCESS_INSTANCE_URL + "/1/operation";
```